README.md

```diff
@@ -67,30 +67,36 @@ curl -Ln -bj "http://localhost:3000/C1233800302-EEDTEST/ogc-api-coverages/1.0.0/
 
 We recommend using [harmony-py](https://github.com/nasa/harmony-py) and its example notebook when working with Harmony.
 
+### Configuration of Services and Supported Collections
+
+Harmony configures each backend service along with its supported collections and their associated UMM-S, UMM-Var records in a file named `services.yml`.
```

Comment:
There is no UMM-Var information in services.yml.

Reply:
See line 529. There is UMM-Var info for `gesdisc/giovanni` service.